homeassistant/components/caldav/calendar.py

```diff
@@ -199,21 +199,47 @@ def update(self):
             dt.start_of_local_day(), dt.start_of_local_day() + timedelta(days=1)
         )
 
+        # Create new vevents for each recurrence of an event that happens today.
+        # For recurring events, some servers return the original event with recurrence rules
+        # and they would not be properly parsed using their original start/end dates.
+        vevents = [event.instance.vevent for event in results]
+        new_vevents = []
+        for vevent in vevents:
+            rrules = vevent.getrruleset()
+            if rrules:
```

Comment:
Please use a guard clause. `if not rrules: continue`

Reply:
I don't understand. What will this accomplish? Both `none` and `[]` evaluate to `False`, and we only want to run that code to create new vevents if it's a recurring event anyway. Is this more a readability/formatting issue?

Reply:
readability/formatting.